package.json

```diff
@@ -31,7 +33,9 @@
     "@wdio/globals": "^9.1.2",
     "@wdio/mocha-framework": "^9.1.2",
     "@wdio/spec-reporter": "^9.1.3",
+    "addons-linter": "^7.3.0",
     "eslint": "^9.13.0",
+    "eslint-config-prettier": "^9.1.0",
```

Comment:
while resolving merge conflicts had to remove package-lock, which exposed that `eslint-config-prettier` is a peer dependency of `eslint-plugin-prettier` and has to be explicitly listed